lib/index-hints.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+/**
+ * An enum of index hints to be used in mysql for querying with index hints
+ *
+ * @property USE
+ * @property FORCE
+ * @property IGNORE
+ */
+const IndexHints = module.exports = { // eslint-disable-line
```

Comment:
Please use `exports.IndexHints = { ... }`

Reply:
Updated in 37772bcb0c0a7bc9eb1982466ce0f9a4dbe341f0